docs/tutorials/projects/pet-treat-dispenser.md

```diff
@@ -243,9 +244,9 @@ Head over to the [**DATA** page](https://app.viam.com/data/view) and select an i
 After selecting the image, you can type a custom tag for some of the objects you see in the image.
 The first thing you want to consider is what tags you are trying to create and how you want your custom model to function.
 
-![The images tab of the data page with a number of images of the dog.](../../img/pet-treat-dispenser/app-data-images.png)
+![The images tab of the data page with a number of images of the dog.](../../img/pet-treat-dispenser/app-data-images-filters.png)
 
-For the treat dispenser, you can tag images with the name of the pet.
+For the treat dispenser, you can tag images with the name of the pet and also tag images that do not include the pet.
```

Comment:
Would be worth pointing out that it's not necessary to have a "not-toast" since any of your filtered images without that tag will automatically be cast as unknown/background.

So in order to have a well-rounded dataset, you could either
* Filter down to your intended images by tags ('toast', 'not-toast') then only select 'toast' when training
* Filter down to tags + whatever other filter combination, like timestamp ('toast' + all images collected that day), then only select 'toast' when training